package.json

```diff
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "start": "node lib/index",
-    "test": "lab -va must",
+    "test": "TRUSTED_SELF_SIGNED=true REGISTER_MEDIATOR=false lab -va must",
```

Comment:
@bradsawadye Why did you set this variable here? I think it would better be set with in the test file to make it more visible to the tester? Setting the ENV variables in the package.json seems liek it's hiding the variable a bit
@BMartinos What do you think?

Reply:
We shouldn't be setting this ENV variable within package.json itself but instead within the file that is testing it/making use of it. This makes the test file more explicit in what it testing and we can easily see the ENV in which it is being tested

Reply:
fixed in https://github.com/jembi/facility-cache/pull/7/commits/d61e6da6cb2d8bb770ee42340fdb2f0a98567098